docs/contribute/api-docs-contribution/guidelines.md

```diff
@@ -0,0 +1,575 @@
+
+Elastic APIs are generally complex, and summaries alone are not enough to explain their purpose and usage.
+Descriptions enable you to add detailed explanations of your OpenAPI objects.
+
+They are essential for transforming machine-readable information into nicely formatted, human-readable prose. You can add paragraph breaks, links, or lists to your descriptions.
+
+Here are some principles for effective descriptions:
+
+- **Explain purpose and impact:** What does this operation/parameter do and why would users need it?
+- **Provide context:** What are the prerequisites or related concepts users should know?
+- **Use formatting:** Use paragraph breaks, lists, and other formatting options to make descriptions readable
+- **Reference related operations:** [Link](#add-links) to complementary APIs or prerequisite steps
+- **Provide usage guidance:** How should users typically use this parameter or operation?
+- **Document constraints:** What are the valid values, formats, or limitations?
+- **Explain data relationships:** For list parameters, clarify how multiple values are handled (comma-separated, arrays, etc.)
+- **Document special formats:** Include expected formats for timestamps (ISO-8601), patterns for wildcards, etc.
+- **Ensure comprehensive coverage:** Ensure all parameters, tags, and information sections include clear descriptions
+- **Use inclusive language:** Avoid problematic terminology (blacklist, whitelist, execute, kill, etc.) and use inclusive language throughout
+
+#### Description examples
+
+::::{tab-set}
+:group: implementations
+
+:::{tab-item} OpenAPI
+:sync: openapi
+
+**Operation descriptions** 
+
+Add descriptions to operation definitions:
+
+```yaml
+paths:
+  /books:
+    get:
+      summary: List books
+      description: |
+        Retrieves a paginated list of books from the library catalog.
+        Use query parameters to filter by author, genre, or publication year.
+        Results are sorted by title by default.
+```
+
+**Parameter descriptions**
+
+Add descriptions to parameter definitions:  
+
+```yaml
+parameters:
+  - name: author
+    in: query
+    description: |
+      Filter books by author name. Supports partial matching - 
+      searching for "smith" will find "John Smith" and "Jane Smithson".
+    schema:
+      type: string
+      example: "Jane Austen"
+```
+
+**Schema property descriptions**
+
+Add descriptions directly to schema properties:
+
+```yaml
+components:
+  schemas:
+    Book:
+      type: object
+      properties:
+        publishedDate:
+          type: string
+          format: date
+          description: |
+            The date when the book was first published in ISO 8601 format.
+            Used for filtering and sorting operations.
+```
+::{tip}
+Learn more about providing long descriptions in YAML in the [OpenAPI docs](https://learn.openapis.org/specification/docs.html#providing-long-descriptions-in-yaml).
```

Comment:
This tip doesn't seem to have rendered properly in my preview. I think maybe it has to be at least three colons, so you might need to increase the number of colons in the other tab annotations, etc

> Learn more about providing long descriptions in YAML in the ...